UnchainedLauncherCore/src/API/A2S.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Sockets;
+using System.Buffers.Binary;
+
+namespace UnchainedLauncher.Core.API
+{
+    public record A2S_INFO (
```

Comment:
Can the casing of this be changed to match the other record types?

Reply:
I think this was left over from when I was matching what it was called in the spec. Fixed it now tho.